_i18n/en/general/privacy.md

```diff
@@ -1,57 +1,82 @@
-AntennaPod is developed and maintained by individual volunteers and not represented by any legal entity. The community doesn't need your data, so the app and website are designed to be fully GDPR compliant. Read on to learn more.
+AntennaPod is developed and maintained by individual volunteers and is not represented by any legal entity. The community has no interest in your data, so the app and website are designed to be GDPR compliant. Read on to learn more.
 
-## What data the AntennaPod app might collect, store and process
-The app only records data strictly necessary for the app to function. This includes for example the list of podcasts you follow, the app settings, and authentication data provided for specific podcasts or services (such as synchronization services). The app does not include any advertisement libraries or any 3rd party tracking (analytics) code, such as Google Analytics.
+## The app
 
-**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not send or upload your data anywhere, except where strictly necessary for the functioning of the app.** Please see the second next section for the cases in which your data may be provided to third parties.
+### Data AntennaPod may collect, store, and process
+The app only records data strictly necessary for the app to function. This includes the list of podcasts you follow, the app settings and authentication data provided for specific podcasts or services (such as synchronization services). The app does not include any advertising libraries or third-party tracking or analytics code, such as Google Analytics.
 
-The only case where your data from 3rd parties is handled, is when using a synchronization service if enabled via the settings. In that case, the app may receive subscriptions and playback events provided by the service.
+**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not send or upload your data anywhere, except where strictly necessary for the functioning of the app or when you actively share debugging information with us.** The next section explains when your data may reach third parties. Details on [app permissions](/documentation/general/app-permissions) are available in our documentation.
```

Comment:
I don't think a legal document should link to other places that could be updated independently

Reply:
I see where you're coming from. But I see the privacy policy not just as a legal document, but also an info page for our users. And I'm that context I think it makes sense to link to related content, even if that content has no legal value.

Maybe we can rephrase such that it's clearer that it's just 'for information '.